server/app/views/questiontypes/CheckboxQuestionFragment.html

```diff
@@ -0,0 +1,38 @@
+<div th:fragment="checkbox (question, messages)" th:with="questionId=${#strings.randomAlphanumeric(8)}" th:id="${questionId}">
+    <fieldset class="usa-fieldset" th:aria-describedby="${questionId} + '-description'" th:with="multiSelectQuestion=${question.createMultiSelectQuestion()}">
```

Comment:
Do you have opinions about using the `*{...}` syntax for anywhere below here that references `multiSelectQuestion`?

It seems nice to avoid repetition, but also like it could be hard to follow

Reply:
I can go either way on that. I have no objections to the * syntax, especially when reusing something. The only thing that's confusing here is that we're using both question and checkboxQuestion, but it's honestly a bit confusing either way...

Reply:
Alright I played with this some more and I find that *{} is limiting in instances like this. You can't pass any arguments when using * syntax, so you can't really convert:
${multiSelectQuestion.getValidationErrors().get(question.getContextualizedPath())} OR 
${multiSelectQuestion.optionIsSelected(option)}

so for those you have to do:
${#object.optionIsSelected(option)} instead, and then you're mixing *s and #objects and it all ends up more muddled. It's probably good in other cases though.

Reply:
Mmmm yeah that's annoying if it's mixed.

I can imagine refactors to make it work nicer, like having the selection status on the option itself so we don't have to ask the question if it's the selected option, but then that has other downsides.